pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
   "extension-helpers",
-  "numpy>=2.0.0rc1,<2.3",
+  "numpy>=2.0.0rc1",
```

Comment:
Why was there a <2.3 in the first place?

Reply:
numpy suggested to use that on release branches and I was going to backport this but then i thought, who cares what numpy thinks.